src/jubjub/mod.rs

```diff
@@ -330,6 +330,8 @@ impl JubjubBls12 {
 
         // Create the 3-bit window table lookups for fixed-base
         // exp of each base in the protocol.
+        // The base g_i of the i'th window is (8^i)*gen
```

Comment:
I seem to recall seeing somewhere that it was `[16^i] gen`. I guess that matches up if we store 8 values and then conditionally negate.

Reply:
It is ``[16^i] gen``, and the comment should be changed.

Reply:
Will switch to [x]. I don't see why it's `[16^i] gen`. 
First of all, it *should* be 8^i when the window size is 3.
And we see in the code below `gen` is added to itself 8 times
to get the new `gen` for the next iteration (line 350: `gen = g`)